amplify/backend.ts

```diff
@@ -2,7 +2,24 @@ import { auth } from "@/amplify/auth/resource";
 import { data } from "@/amplify/data/resource";
 import { defineBackend } from "@aws-amplify/backend";
 
-defineBackend({
+const backend = defineBackend({
   auth,
   data,
 });
+
+backend.auth.resources.cfnResources.cfnUserPool.schema = [
+  {
+    name: "institution",
```

Comment:
We can get rid of all of this because we store it in DynamoDB. If you get an error, simply delete all your sandbox resources andredeploy from scratch. (User Attributes can not be deleted from a UserPool)